scripts/load_csi_provisioners.sh

```diff
@@ -14,10 +28,13 @@ EOT
 
 echo "var CSIDriverList = []*CSIDriver{" >> ${current_directory}/../extra/csi-drivers-temp.go
 while read p; do
-  if [[ $p == [* ]];
-  then
+  if [[ $p == [* ]]; then
     IFS='|'
     read -a fields <<< "$p"
+    if [[ ${#fields[@]} -ne 8 ]]; then
```

Comment:
This check is ignoring some of the CSI providers. Some of them are missing the `Features` and are getting ommitted

Reply:
I totally missed that the features column was treated as optional. Thanks for catching that.

Changing the check to `if [[ ${#fields[@]} -lt 7 ]]; then`, so as to not accidentally skip legit CSI entries.

And, adding an extra check for the features field (to avoid unset variable error)
```
if [[ ${#fields[@]} -ge 8 ]]; then
  features=$(sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//' <<<${fields[7]})
fi
```

Please let me know if you have a differing opinion on this!

Reply:
Resolving conversation - I believe I have addressed the problem with the latest commit https://github.com/kastenhq/kubestr/pull/72/commits/a4f64dfb4fc3f780d9d20d3a1af3039d544bbfc4

Please reopen if required.